sdk/core/core-amqp/README.md

```diff
@@ -55,7 +55,7 @@ You can run the samples by cloning the repo or copy pasting the below sample in
 ```
 
 The samples below are generic for EventHubs and Servicebus. You can find EventHub specific samples,
-in the [samples](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.5/sdk/core/amqp-common/samples) directory.
+in the [samples](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/core-amqp_1.0.0-preview.1/sdk/core/core-amqp/samples) directory.
```

Comment:
The updated link here won't work. The previous link used a "tag", we should use the "master" branch instead